Bottom: 7ac47347c15127f9310ac2faaf1acf7bb4dbe2bd
Top:    4708e0e756ad1dd598a75e5ce77692840118112f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-11 23:47:42 +0000

Remove unused HistorySize field from the settings structure


---

diff --git a/src/global.h b/src/global.h
index 3794448..19f4454 100644
--- a/src/global.h
+++ b/src/global.h
@@ -668,8 +668,7 @@ typedef struct			/* some resources... */
     /* connections is done */
     AutoPlace;			/* flag which says we should force placement of the
 				   windows on startup */
-  int HistorySize,		/* FIXME? Used in hid/xaw code only. */
-    init_done;
+  int init_done;
 }
 SettingType, *SettingTypePtr;
